src/rpc/handlers/LedgerEntry.hpp

```diff
@@ -302,6 +304,7 @@ class LedgerEntryHandler {
                   meta::WithCustomError{modifiers::ToNumber{}, Status(ClioError::rpcMALFORMED_ORACLE_DOCUMENT_ID)}},
              }}},
             {JS(ledger), check::Deprecated{}},
+            {"include_deleted", validation::Type<bool>{}},
```

Comment:
I assume you checked and there is no `JS(include_deleted)` but thought i'd ask ❓

Reply:
Yes there's no JS(include_deleted)